### PR TITLE
 NOISSUE - Add searchable Channels name 

### PR DESCRIPTION
**Pull request by manuio**

### CODE REVIEW

File: things/api/http/responses.go [hunk 1]
```diff
@@ -219,4 +219,5 @@ type pageRes struct {
 	Total  uint64 `json:"total"`
 	Offset uint64 `json:"offset"`
 	Limit  uint64 `json:"limit"`
+	Name   string `json:"name"`
```
Comment (anovakovic01) [inline things/api/http/responses.go]: Why do you return the name in the response?

File: things/postgres/channels_test.go [hunk 2]
```diff
@@ -41,13 +42,22 @@ func TestChannelSave(t *testing.T) {
...
-			desc: "create invalid channel",
+			desc: "create channel with invalid ID",
 			channel: things.Channel{
 				ID:    "invalid",
 				Owner: email,
 			},
 			err: things.ErrMalformedEntity,
 		},
+		{
+			desc: "create thing with invalid name",
```
Comment (anovakovic01) [inline things/postgres/channels_test.go]: `channel`, not `thing`

File: things/service_test.go [hunk 3]
```diff
@@ -535,10 +536,26 @@ func TestListChannels(t *testing.T) {
 			size:   0,
 			err:    things.ErrUnauthorizedAccess,
 		},
+		"list with existent name": {
```
Comment (anovakovic01) [inline things/service_test.go]: existing

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**